website/docs/reference/options.mdx

```diff
@@ -322,6 +322,15 @@ Type: `http.Handler`
 
 #### Middleware
 
+:::info
+
+This does not work with vite v5.0.0+ and wails v2. 
```

Comment:
Do you think we should say that this is due to vite changing what it does in v5 vs v4?

Reply:
indeed, new commit to highlight changes in vite as the root cause